jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -44,8 +47,9 @@ class Slice(PluginTemplateMixin):
     * ``show_value``
       Whether to show slice value in label to right of indicator.
     """
-    _cube_viewer_cls = CubevizImageView
-    _cube_viewer_default_label = 'flux-viewer'
+    _cube_viewer_cls = (CubevizImageView, RampvizImageView)
+    _cube_viewer_default_label = None  # must be filled in by helper on initialization
```

Comment:
I really don't like the helper being responsible for setting something in the plugin, can this be retrieved from the helper during init instead?  Or would a subclass here as well help?